model-optimizer/mo/utils/versions_checker.py

```diff
@@ -30,6 +30,22 @@ def check_python_version():
         return 1
 
 
+def get_imported_module_version(imported_module):
+    """
+    Get imported module version
+    :return: version(str) or raise AttributeError exception
+    """
+    version_attrs = ["__version__", "VERSION", "version"]
+    installed_version = None
+    for attr in version_attrs:
+        if installed_version is None:
+            installed_version = getattr(imported_module, attr, None)
+            installed_version = installed_version if isinstance(installed_version, str) else None
+        else:
+            return installed_version
+    raise AttributeError("{} module doesn't have version attribute".format(imported_module))
```

Comment:
I wouldn't raise an exception because I don't want to affect customers for which this function mayn't work out. Let us continue to return `None`, warning is not so terrific.
Let us reimplement as so:
```
    for version_attr in ["__version__", "VERSION", "version"]:
        installed_version = getattr(imported_module, version_attr, None)
        if isinstance(installed_version, str):
            return installed_version
    return None
```

Reply:
@rkazants I use a raise exception, because you have the check for exceptions below https://github.com/openvinotoolkit/openvino/pull/7041/files#diff-432e36d3b7df2d162c49ed64502e5c3dca92a1cd41d46bc565c13176941e9a0aL261

Reply:
you are correct. Pls re-implement it due to possible incorrect behavior in the last iteration as we agreed.